gauge-java.go

```diff
@@ -297,19 +296,22 @@ func printUsage() {
 	os.Exit(2)
 }
 
-func runCommand(cmdName string, args []string) {
-	cmd := runCommandAsync(cmdName, args)
+func runJavaCommand(cmdName string, args []string, classpath string) {
+	cmd := runJavaCommandAsync(cmdName, args, classpath)
 	err := cmd.Wait()
 	if err != nil {
 		fmt.Printf("process %s with pid %d quit unexpectedly. %s\n", cmd.Path, cmd.Process.Pid, err.Error())
 		os.Exit(1)
 	}
 }
 
-func runCommandAsync(cmdName string, args []string) *exec.Cmd {
+func runJavaCommandAsync(cmdName string, args []string, classpath string) *exec.Cmd {
 	cmd := exec.Command(cmdName, args...)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
+	env := os.Environ()
+	env = append(env, fmt.Sprintf("CLASSPATH=%s", classpath))
+	cmd.Env = env
```

Comment:
@mdolinin If the `CLASSPATH` env already exists then it should overwrite it but currently it will append multiple `CLASSPATH` to environment.  You can use `os.Setenv` to add CLASSPATH env and `exec.Command` will inherit that.